Makefile

```diff
@@ -61,6 +81,28 @@ start-openshift-with-catalog:
 start-openshift-with-catalog-and-ansible-service-broker: start-openshift-with-catalog install-ansible-service-broker
 .PHONY: start-openshift-with-catalog-and-ansible-service-broker
 
+prepare-remote-openshift: clean-remote-openshift
```

Comment:
I believe we should also do this step for OpenShift local. Even though it's a bit redundant, we should simplify and unify makefiles this way.

Reply:
+1, lets keep the amount of repetition to a minimum where possible.